registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -214,6 +215,7 @@
 - name: clusterctl
   dmap:
     "sha256:6cb237a6445f64afb7ed92517c79fcf3a63b02e20a28aea3fc4274f9aeffd6d1": ["v1.10.0-beta.0"]
+    "sha256:0bd716555d228c055bc2210b81e3276575d989a8f12c60060da269360536aab9": ["v1.10.0-beta.1"]
```

Comment:
`0bd716555d22` matches the staging repo: https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:0bd716555d228c055bc2210b81e3276575d989a8f12c60060da269360536aab9/details?inv=1&invt=Abs_9w 👍🏻